src/Oro/Bundle/RequireJSBundle/DependencyInjection/Compiler/CacheProviderPass.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Oro\Bundle\RequireJSBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+
+/**
+ * This compiler pass replaces the service definition of the cache provider
+ * for the RequireJS configuration with the Doctrine PhpFileCache if the
+ * OroCacheBundle is not available.
+ *
+ * @author Stefano Arlandini <sarlandini@alice.it>
+ */
+class CacheProviderPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        if (!$container->hasDefinition('oro.cache.abstract')) {
+            $definition = new Definition('Doctrine\\Common\\Cache\\PhpFileCache', array('%kernel.cache_dir%/oro_data'));
```

Comment:
I would like to do it, but Symfony 2.7 (which is declared to be the minimum version supported for this bundle in `composer.json`) runs on PHP 5.3+ while the `::class` syntax has been available since PHP 5.5